packages/host/src/lib/host.ts

```diff
@@ -72,6 +76,11 @@ export class Host implements IComponent {
      */
     logger: Logger;
 
+    /**
+     * Instance of class providing logging utilities.
+     */
+    objLogger: ObjLogger;
```

Comment:
Would `logger` be still used for something, e.g. `/log` endpoint or we plan to get rid of it entirely?

Reply:
get rid. `/log` now provides logs in nd-json format